content/security/improvements.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: security
+title: "Improvements by the Security Team"
+section: security
+---
+
+These are some contributions by members of the Jenkins security team that weren't delivered as security fixes, but still are security-related.
+
+
+== 2020
+
+Jenkins (core)::
+- Plugin Manager: https://github.com/jenkinsci/jenkins/pull/4553[Add inline security warnings to installed plugins list]
+- Plugin Manager: https://github.com/jenkinsci/jenkins/pull/4513[Show on 'updates' tab when a warning would be fixed]
```

Comment:
Optional, and possibly ill-advised to include this, since the links will eventually be broken when those releases are no longer displayed in the changelog

```suggestion
- Plugin Manager: https://github.com/jenkinsci/jenkins/pull/4513[Show on 'updates' tab when a warning would be fixed] - link:/changelog/#v2.224[Jenkins 2.224]
```